### PR TITLE
Automerge, actual live transcription

### PR DESCRIPTION
**Pull request by pajowu**

### CODE REVIEW

**Comment by pajowu** [comment]
This is currently a stub pr, I'll update it with the code I already have later

File: worker/run.py [hunk 1]
```diff
@@ -21,11 +21,18 @@ async def main():
         help="url to the task coordinator (aka the transcribee backend)",
         default="http://localhost:8000",
     )
+    parser.add_argument(
```
Comment (anuejn) [inline worker/run.py:24]: why is this seperate?

Comment (anuejn) [reply worker/run.py:24]: and can I have a default that picks the coordinator?

**Review by anuejn — COMMENTED** [review]
Seems good to me :)

**Review by anuejn — APPROVED** [review]

**Pull request merged**